v2g-liberty/rootfs/root/appdaemon/tests/test_settings_manager.py

```diff
@@ -276,6 +276,36 @@ def test_upgrade_schedule_settings_initialised(self, settings_manager):
             settings_manager.get("input_boolean.schedule_settings_initialised") is True
         )
 
+    @patch("os.path.exists", lambda _: True)
+    def test_upgrade_fm_url(self, settings_manager):
```

Comment:
```suggestion
    def test_upgrade_fm_url(self, settings_manager):
        """Check that the url to Seita's FlexMeasures server is updated automatically (update from March 2025)."""
```

Reply:
Good to mention the date. I'll also add that this in the end (a years time from now) could be removed, both the original coe and the test.

Reply:
I did find one instance where it was not correctly upgraded but i have the impression your stored search looked in a different branch as it found many that where already changed.